README.adoc

```diff
@@ -4950,6 +4950,28 @@ LOREM
 "when an unknown printer took a galley of type and scrambled it to make a type specimen book."
 ----
 
+== Enumerable
```

Comment:
Might be good to check if something else can't be moved under this section as well, as it's kind of weird to have a section with a single entry in it.